CHANGELOG.rst

```diff
@@ -14,6 +14,7 @@ Added
 Changed
 ~~~~~~~
 
+* Improved instructions in requirements.txt and dist_utils.py comment headers (improvement) #4774
```

Comment:
Let's please follow one consistent way of composing changelog: from top to bottom when newer records go to the end like you did for another "fix" changelog, and not vice versa.

Reply:
```suggestion
* Improved development instructions in requirements.txt and dist_utils.py comment headers (improvement) #4774
```